### DISCUSSION TITLE
Remove pp changes by reason use NoFail mode

### DISCUSSION THREAD

**Discussion by o-200** [ppy/osu#18424] [Ideas] [answered]
I mean a group of players using NoFail for downgrades they total pp on profile.

https://osu.ppy.sh/users/9823143

When you're have chokes or misser with nf your pp counter on score is divided by 10%.

it allows players to play in tournaments, where others in fact have less pp, than you actually have.

**Comment by o-200** [comment]
I used my profile as an example, but they doesnt have a lot of "choked" scores. I checked my profile with a bot that doesn't consider nofail. For 6500pp Nofail profile = 6900pp "clear" profile. 

Despite the fact that I've been playing with nofail for 1 year, the difference is not very big.

**Reply by uzervlad** [reply]
> For 6500pp Nofail profile = 6900pp "clear" profile.

Not sure where you got those numbers from
![image](https://user-images.githubusercontent.com/24654479/170525162-fdb44369-4cc9-4516-82e7-54bf89459605.png)

**Reply by o-200** [reply]
my fault.

then you should to check https://osu.ppy.sh/scores/osu/4106053113

by the calc that score give me 380-410 pp. But with 1x50 +nf that just a 309pp.

**Reply by smoogipoo** [reply]
I'm not sure where you're getting these numbers from, but they're not accurate. I'd advise you not to use random websites to calculate your score's PP because they usually don't track the latest changes accurately, and it sounds like that calculator doesn't have the changes that were applied in https://github.com/ppy/osu-performance/pull/127.

You need to use https://github.com/ppy/osu-tools/ for the latest changes.

Inclusive of all the changes since the last live version (i.e. pp values will differ slightly from the website):
```
0x50 +NM: 309.48 (50 added into 300)
1x50 +NF: 299.76
```

From that user, the most misses I could find with +NF was https://osu.ppy.sh/scores/osu/4055535839, at 5. This is the only score that would have a 0.9 PP multipler, but that's only ~30pp for this score. All other scores are 1 or 2 misses, and would have a pp multiplier of 0.96 or 0.98 which is not a big deal - https://osu.ppy.sh/scores/osu/3900930325 goes from 306pp to 318pp for comparison.

**Comment by bdach** [comment, marked as answer]
has been rejected several times. removing the pp penalty for playing nofail would make it so that *everyone* would optimally put no fail on because of the lack of penalty.

duplicate of: https://github.com/ppy/osu/issues/9844, https://github.com/ppy/osu/discussions/14114, https://github.com/ppy/osu/discussions/16779

i believe that deranking via nofail will be addressed in upcoming restructurings of the score/pp system, by removing the cases where a new play overwrites an old play leaderboard-wise despite being worth less pp.

**Reply by smoogipoo** [reply]
It's been relaxed a bit since https://github.com/ppy/osu-performance/pull/127, you now need at least 5 misses to get to the 0.9 factor. We could consider relaxing it even further in the future.
